manta-parameters/src/lib.rs

```diff
@@ -47,9 +110,10 @@ pub mod github {
     pub const CRATE: &str = "manta-parameters";
 
     /// Default GitHub Branch
-    pub const DEFAULT_BRANCH: &str = "main";
+    pub const DEFAULT_BRANCH: &str = "feat/mantapay-v1";
```

Comment:
move to `main`

Reply:
I'll leave this for the last commit, so the CI checks the compatibility tests before giving them the ignore flag.